pdns/recursordist/docs/lua-scripting/hooks.rst

```diff
@@ -24,8 +24,13 @@ It also means that to print such names, the ``:toString`` method must be used (o
 Once a script is loaded, PowerDNS looks for the interception functions in the loaded script.
 All of these functions are optional.
 
-If a function returns true, it will indicate that it handled a query.
-If it returns false, the Recursor will continue processing unchanged (with one minor exception).
+If ``ipfilter`` returns ``true``, the query is dropped.
+If ``preresolve`` returns ``true``, it will indicate it handled a query, and the recursor will send to result as constructed in the functions to the client.
+If it returns ``false``, the Recursor will continue processing.
+For the other functions, the return value will indicate if DNSSEC validation should take place.
```

Comment:
While it's technically the only thing that changes today when `true` is returned, I'm not sure we should document it that way. Perhaps:
```suggestion
For the other functions, the return value will indicate that an alteration has been made. In that case DNSSEC validation will be automatically disabled since the content might not be genuine anymore.
```